typedjsonrpc/registry.py

```diff
@@ -124,6 +134,7 @@ def _handle_exceptions(self, method, is_notification=False, msg_id=None):
                     debug_url = self._store_traceback()
                 else:
                     debug_url = None
+                self._logger.exception(exc_info[2])
```

Comment:
After change, it logs the message id, debug url, and traceback. And the method name is shown within the traceback. I didn't have to put exc_info in the exception method explicitly. In fact, Exception info is always added to the logging message.
https://docs.python.org/2/library/logging.html#logging.Logger.exception